backend/coins/btc/account_test.go

```diff
@@ -90,7 +90,7 @@ func TestAccount(t *testing.T) {
 
 	transactions, err := account.Transactions()
 	require.NoError(t, err)
-	require.Equal(t, []*accounts.TransactionData{}, transactions)
+	require.Equal(t, accounts.OrderedTransactions{}, transactions)
```

Comment:
Would be nice to have a test with 2 entries where it verifies the sorting was done, in correct order.